src/components/Error/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import * as styles from "./Error.module.scss";
+
+export default function Error({ errorWord, word }) {
+  return (
+    <>
+      {errorWord.split("").map((errorLetter, index) => {
+        return (
+          <span
+            key={index}
+            className={
+              word[index].toUpperCase() !== errorLetter ? styles.incorrect : ""
```

Comment:
I think this is passable, but it's a little confusing to read. `errorLetter` isn't necessarily an error. So when you say `word[index] does not equal errorLetter apply 'incorrect' styles to it` and that's the opposite of what you want to happen. 

Again this is passable, and I know what you're going for, but you have to reverse engineer the code to understand it.